include/rrc/io/CommandLine.h

```diff
@@ -0,0 +1,36 @@
+/**
+ *  @autor Nikita Somenkov
+ *  @date 08/10/2016
+ */
+#pragma once
+
+#include "IODevice.h"
+
+namespace rrc {
+
+    template <typename Implementation>
+    class CommandLine_ : public IODevice
+    {
+    public:
+        CommandLine_(IOService &service, Console console);
+
+        virtual void open();
+        virtual bool isOpen();
+
+        virtual void close();
+
+        virtual void read(IOBufferPtr buffer, Callback &callback);
+        virtual void write(const IOBufferPtr buffer, Callback &callback);
```

Comment:
If you're overriding virtual method you should use "override" keyword in the end of method signature.